src/store/apps.ts

```diff
@@ -18,25 +18,25 @@ export type app = {
   /** A One line description of the app (max 50 characters) */
   tagline: string;
   /** A longer description of the app (50 to 200 words) */
-  description: string;
+  description?: string;
   /** The person(s) who created the app */
   developer?: string;
   /** The person(s) who created the app */
   developers?: Record<string, string>;
   /** The dependencies of the app */
-  dependencies: (Dependency | Dependency[])[];
+  dependencies?: (Dependency | Dependency[])[];
   /** The url to the app's Git repository */
   repo: string | Record<string, string>;
   /** The url to the app's support website/chat */
```

Comment:
Actually, please revert this commit and cherry-pick https://github.com/runcitadel/dashboard/pull/43/commits/387beac254388021cff637ffc308ebc684fd183f instead, it is a cleaner solution.

Reply:
There is still a minor bug in the SDK types where the id is missing, so the solution in that commit is probably a better solution and also has cleaner code.